beeline/pom.xml

```diff
@@ -197,6 +197,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <version>3.1.1</version>
```

Comment:
I don't think you need to include an explicit version number here.

Reply:
Thanks David. These changes are not final. I am having a new team member look at these changes. There is more work to be done in terms of building a beeline standalone distributable (.zip file). I am just transitioning these code changes for him to take a look.

But your comment is noted.